src/main/java/telraam/logic/positioner/nostradamus/Nostradamus.java

```diff
@@ -0,0 +1,154 @@
+package telraam.logic.positioner.nostradamus;
+
+import org.jdbi.v3.core.Jdbi;
+import telraam.database.daos.BatonSwitchoverDAO;
+import telraam.database.daos.StationDAO;
+import telraam.database.daos.TeamDAO;
+import telraam.database.models.BatonSwitchover;
+import telraam.database.models.Detection;
+import telraam.database.models.Station;
+import telraam.database.models.Team;
+import telraam.logic.positioner.Position;
+import telraam.logic.positioner.PositionSender;
+import telraam.logic.positioner.Positioner;
+
+import java.util.*;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+public class Nostradamus implements Positioner {
+    private static final Logger logger = Logger.getLogger(Nostradamus.class.getName());
+    private final int INTERVAL_CALCULATE_MS = 500; // How often to handle new detections (in milliseconds)
+    private final int INTERVAL_FETCH_MS = 60000; // Interval between fetching baton switchovers (in milliseconds)
```

Comment:
ik denk dat dit wel vaker mag gebeuren? of zorgt dat ergens voor problemen?

Reply:
Shorten the interval [27d1dc4](https://github.com/12urenloop/Telraam/pull/142/commits/27d1dc4055b92d194436a531b12b30fe1d82eda1)